android/src/main/java/com/otpless/ionic/OtplessPlugin.java

```diff
@@ -242,6 +249,76 @@ public void showPhoneHintLib(PluginCall call) {
         });
     }
 
+    @PluginMethod
+    public void attachSecureSDK(PluginCall call) {
+        String appId = call.getString("appId", "");
+        try {
+            Class<?> managerClass = Class.forName("com.otpless.secure.OtplessSecureManager");
+            Object managerInstance = managerClass.getField("INSTANCE").get(null);
+            Method creatorMethod = managerClass.getDeclaredMethod("getOtplessSecureService", Activity.class, String.class);
+            OtplessSecureService secureService = (OtplessSecureService) creatorMethod.invoke(managerInstance, getActivity(), appId);
+            otplessView.attachOtplessSecureService(secureService);
+            call.resolve();
+        } catch (ClassNotFoundException | NoSuchMethodException ex) {
```

Comment:
can be wrapped in single 
catch(ClassNotFoundException | NoSuchMethodException | IllegalAccessException | InvocationTargetException | NoSuchFieldException)